src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleasesController.cs

```diff
@@ -107,7 +107,9 @@ public async Task<ActionResult<List<DataFileInfo>>> ReorderDataFiles(Guid releas
         [RequestFormLimits(ValueLengthLimit = int.MaxValue, MultipartBodyLengthLimit = int.MaxValue)]
         public async Task<ActionResult<DataFileInfo>> UploadDataSet(Guid releaseVersionId,
             [FromQuery(Name = "replacingFileId")] Guid? replacingFileId,
-            [FromQuery(Name = "title")] string title,
+            [FromQuery(Name = "title")]
+            [MaxLength(120, ErrorMessage = "Subject title must be 120 characters or less")]
```

Comment:
I think `Title` on its own would be more appropriate here. We've been trying to phase out usage of the term 'Subject' for a while now.

Same comment applies to the annotation on `UploadDataSetAsZip`.

Reply:
See comments about matching front end field names.